java/common/src/main/java/org/ray/util/LambdaUtils.java

```diff
@@ -0,0 +1,31 @@
+package org.ray.util;
+
+import java.io.Serializable;
+import java.lang.invoke.SerializedLambda;
+import java.lang.reflect.Method;
+
+/**
+ * see http://cr.openjdk.java.net/~briangoetz/lambda/lambda-translation.html.
+ */
+public final class LambdaUtils {
+
+  private LambdaUtils() {
+  }
+
+
+  public static SerializedLambda getSerializedLambda(Serializable lambda) {
```

Comment:
Let's also use `RayFunc` as the parameter type here.

Reply:
MethodId is not only for RayFunc, it clould used for all lambda call. the check of RayFunc has add in entrance